src/details/MovieDetailsCard.js

```diff
@@ -29,7 +26,7 @@ class MovieDetailsCard extends Component {
             <p>{this.state.movie.overview}</p>
             <p>{this.formatGenreString(this.state.movie.genres) + ' * ' + this.state.movie.release_date + ' * ' + this.formatRuntimeString(this.state.movie.runtime)}</p>
             <p>AVERAGE RATING: {this.state.movie.average_rating.toFixed(2)}</p>
-            <table>
+            {!this.state.movie.budget || !this.state.movie.revenue && <table>
```

Comment:
It looks like budget and revenue are no longer displayed on the UI for any of the movies. I'm not sure where this issue is coming into play but wanted to mention it here.